### DISCUSSION TITLE
Will their be an issue

### DISCUSSION THREAD

**Discussion by rishthedev** [ChefKissInc/NootedRed#240] [Q&A] [answered]
I have a ryzen 7 5700G. I am planning to installing sonoma 14.4.1 and use nootedred to get gpu acceleration for my igpu. Will it cause rendering issues. Please can someone confirm

**Comment by SalmonPete** [comment, marked as answer]
Your 5700G will be fine and capable of full acceleration.

**Reply by rishthedev** [reply]
Thanks man

**Comment by darthsian** [comment]
> I have a ryzen 7 5700G. I am planning to installing sonoma 14.4.1 and use nootedred to get gpu acceleration for my igpu. Will it cause rendering issues. Please can someone confirm

NootedRed is much more stable on Ventura than on Sonoma. I would wait to switch to Sonoma for now.

**Reply by SalmonPete** [reply]
In my case 4700G/Renoir Sonoma is more Stable than it was a few weeks ago. The latest updates for NootedRed have seen real improvements, for Renoir at least.

**Reply by rishthedev** [reply]
No bro nootedred works perfectly fine. Thank you for saying that their was no problem. That made my get sonoma. It's a hit or miss. Nootedred may or may not cause issues

**Reply by oceanondawave** [reply]
Mine works fine too. I've updated to the latest version of Sonoma.
